requirements.txt

```diff
@@ -26,14 +28,10 @@ charset-normalizer==3.3.2
     # via requests
 click==8.1.7
     # via pip-tools
-colorama==0.4.6 ; platform_system == "Windows"
```

Comment:
Don't remove this

Reply:
Yeah -- I forgot to go back & fix those up.